### DISCUSSION TITLE
setRowFormat не работает

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#530] [Как это сделать?] [answered]
f1=: setRowFormat(condition: $#nfv != true; background: "Gainsboro") не оказывает никакого влияния на строку, только на ячейку.
Цвет панели задан у некоторых столбцов в таблице, но не у всех...

**Comment by erpic-ru** [comment, marked as answer]
setRowFormat используется в коде форматирования строк (в настройках таблицы), соответственно там должно быть что-то такое: 

`f1=:setRowFormat(condition: #oplata = true; background: "#d9ead3";)`

а не $#nfv

**Reply by ttm-alexeypolunin** [reply]
И я что-то не припом, что бы гейнсборроу относился к цветам html. Вот полный список: https://colorscheme.ru/html-colors.html

Остальные как HEX задаются как Эрпик привел пример.
